qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy

```diff
@@ -90,6 +92,7 @@ class DeploymentEventGraphQLTest extends BaseSpecification {
     private final gqlService = new GraphQLService()
 
     @Category(GraphQL)
+    @IgnoreIf({ Env.CI_JOBNAME.contains("postgres") })
```

Comment:
I recommend we don't skip it.  I'm waiting on a test to pass and then I will put the fix in review.

https://github.com/stackrox/stackrox/pull/3319